main.cpp

```diff
@@ -347,17 +346,9 @@ class Mod : GenericMod {
 		if (creature->entity_data.hostility_type != cube::Creature::EntityBehaviour::Player &&
 			creature->entity_data.hostility_type != cube::Creature::EntityBehaviour::Pet)
 		{
-			//*stat /= creature->entity_data.level + 1;
-			//*stat += m_CreatureScaling.at(type) * (GetCreatureLevel(creature) - LEVELS_PER_REGION / 2);
-			//*stat *= creature->entity_data.level + 1;
-			if (type != STAT_TYPE::HASTE && type != STAT_TYPE::CRIT) {
-				*stat *= 0.2 * std::pow(2.7183, 0.2 * GetCreatureLevel(creature)) / 1.21 * std::pow(0.99, 1 + 0.07 * GetCreatureLevel(creature) * GetCreatureLevel(creature));
-			}
-			else {
-				//*stat /= creature->entity_data.level + 1;
-				//*stat += m_CreatureScaling.at(type) * (GetCreatureLevel(creature) - LEVELS_PER_REGION / 2);
-				//*stat *= creature->entity_data.level + 1;
-			}
+			*stat += m_CreatureScaling.at(type) * GetCreatureLevel(creature);
```

Comment:
Is this meant to be added? Because I meant either remove the function calls as you have done or apply m_CreatureScaling to line `350`. (Basically just putting `m_CreatureScaling.at(type) * 0.2 * .... `  Now you have done both and one the wrong way thus changed the scaling from how you had it before

Reply:
I tried this, but it made high level enemies overpowered